### PR TITLE
Document usage of markdown for remark fields

### PR DESCRIPTION
**Pull request by WouterSioen**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -699,7 +699,7 @@ Get details for a single contact.
                         + id: `40157f9d-1322-4f2e-b245-188efdc52cc2` (string)
             + language: `en` (string)
             + payment_term (PaymentTerm, nullable)
-            + remarks: `First contact at expo` (string)
+            + remarks: `First contact at expo` (string) - Formatted in markdown
```
Comment (jdrieghe) [inline apiary.apib]: I've been using ` - Uses markdown formatting` in my other PR. Maybe we should make it consistent?

 :ok: by @WouterSioen on 12/06/2018 10:25:13

**Review by rathesDot — DISMISSED** [review]
+1 for consistency, though

**Review by jdrieghe — DISMISSED** [review]

**Review by JordyDeGroeve — DISMISSED** [review]

File: src/02-crm/companies.apib [hunk 2]
```diff
@@ -200,7 +200,7 @@ Update a company.
         + bic: `BICBANK` (string, optional, nullable)
         + language: `en` (string, optional, nullable)
         + responsible_user_id: `0ea94804-401d-4dbd-a577-c2d60998f798` (string, optional, nullable)
-        + remarks: `Met at expo` (string, optional, nullable)
+        + remarks: `Met at expo` (string, optional, nullable) - Uses markdown formatting
```
Comment (dietrichm) [inline src/02-crm/companies.apib]: B: Updating a company through API v2 does not parse Markdown syntax yet. It's noted in a code comment and on my TODO list (after I removed the AuthenticatedPerson from the flow).

File: apiary.apib [hunk 3]
```diff
@@ -699,7 +699,7 @@ Get details for a single contact.
                         + id: `40157f9d-1322-4f2e-b245-188efdc52cc2` (string)
             + language: `en` (string)
             + payment_term (PaymentTerm, nullable)
-            + remarks: `First contact at expo` (string)
+            + remarks: `First contact at expo` (string) - Uses markdown formatting
```
Comment (dietrichm) [inline apiary.apib]: PP: use correct capitalisation ("Markdown").

 :ok: by @WouterSioen on 12/06/2018 13:49:07

**Review by dietrichm — DISMISSED** [review]
I'm not sure Markdown is actually parsed everywhere as  stated here. Could we make some time available to check and fix?

File: src/02-crm/companies.apib (hunk 2 above)
Comment (WouterSioen) [inline src/02-crm/companies.apib]: If this doesn't happen, we have a bigger problem then that the documentation is not correct.

If so, this needs to be fixed asap imo.

 :ok: by @WouterSioen on 19/06/2018 09:17:01

**Review by jenssegers — APPROVED** [review]
I'm guessing there's no other way to hint Markdown support other than a comment.

Maybe something like `remarks: 'Met at expo' (Markdown)` could work with a custom datatype. But this is also ok 👌

**Review by jdrieghe — APPROVED** [review]

**Review by dietrichm — DISMISSED** [review]
We still need to fix `companies.update` first. No Markdown is parsed there yet.

**Comment by jdrieghe** [comment]
Can be merged after https://github.com/teamleadercrm/core/pull/5542 is deployed to production

**Review by JordyDeGroeve — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Pull request merged**